### PR TITLE
Docs: Better explanation of `EXCEPT` clause

### PR DESCRIPTION
**Pull request by felix021**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/sql-reference/statements/select/except.md [hunk 1]
```diff
@@ -146,8 +146,73 @@ Result:
...
+
+```sql
+INSERT INTO test1 VALUES ('Felix', NOW()), ('Bob', NOW());
+```
+
+To show the basic usage, you can select all fields except `created_at`:
+
+```sql
+SELECT * EXCEPT(created_at) FROM test1
+```
```
Comment (rschu1ze) [inline docs/en/sql-reference/statements/select/except.md:174]: Please include the result of the SELECT in l. 173 here.

File: docs/en/sql-reference/statements/select/except.md [hunk 2]
```diff
@@ -146,8 +146,73 @@ Result:
...
+INSERT INTO test1 VALUES ('Felix', NOW()), ('Bob', NOW());
+```
+
+A simple `select` query would show all versions:
+```text
+┌─name──┬──────────created_at─┐
+│ Bob   │ 2024-07-03 15:06:38 │
+│ Felix │ 2024-07-03 15:06:38 │
+└───────┴─────────────────────┘
+┌─name──┬──────────created_at─┐
```
Comment (rschu1ze) [inline docs/en/sql-reference/statements/select/except.md:188]: Why do the values in the first and second part have the same timestamps in `created_at`? Are they not created at different times?

Comment (felix021) [reply docs/en/sql-reference/statements/select/except.md:188]: Yeah, they are different, `38`s vs `35` for the seconds, please take a look again?

Comment (felix021) [reply docs/en/sql-reference/statements/select/except.md:188]: But it's not so obvious. I'll update it for better illustration

File: docs/en/sql-reference/statements/select/except.md (hunk 1 above)
Comment (felix021) [reply docs/en/sql-reference/statements/select/except.md:174]: Done. PTAL.

File: docs/en/sql-reference/statements/select/except.md (hunk 2 above)
Comment (felix021) [reply docs/en/sql-reference/statements/select/except.md:197]: Done.

**Comment by rschu1ze** [comment]
@felix021 Thanks for this PR but I noticed that it would duplicate existing documentation of the EXCEPT clause: https://clickhouse.com/docs/en/sql-reference/statements/select#except

**Comment by felix021** [comment]
> @felix021 Thanks for this PR but I noticed that it would duplicate existing documentation of the EXCEPT clause: https://clickhouse.com/docs/en/sql-reference/statements/select#except

OK. Though it's hard to be found in the search result:

image

**Comment by rschu1ze** [comment]
The search function does unfortunately not really work, perhaps because it is provided by some third party vendor.

**Pull request closed without merging**